HW_3/templates/index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>HW_3</title>
+</head>
+<body>
+<h1>Регистрация</h1>
```

Comment:
Обработку `flash` сообщений можно добавить тут
```suggestion
<h1>Регистрация</h1>
    {% with messages = get_flashed_messages() %}
        {% if messages %}
        <ul>
            {% for message in messages %}
            <li>
                    {{ message }}
            </li>
            {% endfor %}
        </ul>
        {% endif %}
    {% endwith %}
```